src/sonic-yang-models/yang-models/sonic-system-port.yang

```diff
@@ -40,13 +40,8 @@ module sonic-system-port {
                 }
 
                 leaf ifname {
```

Comment:
>ifname [](http://example.com/codeflow?start=21&length=6)

If you think this is a bugfix, could you also add testcases?

Reply:
@qiluo-msft, is the not bugfix. It was an extra check which was added for this yang which is not required. The current tests cover the current yang model changes